lib/cocoapods-core/cdn_source.rb

```diff
@@ -144,6 +160,10 @@ def versions(name)
     def specification_path(name, version)
       raise ArgumentError, 'No name' unless name
       raise ArgumentError, 'No version' unless version
+      unless versions(name).include?(Version.new(version))
```

Comment:
This is the added code safe guarding against deleted podspecs?

Reply:
Yes.